File: backend/internal/model/response/account.go
package response

type AccountResponse struct {
	ID             int64     `json:"id"`
	AccountID      string    `json:"account_id"`
	Customer       string    `json:"customer"`
	CustomerID     int64     `json:"customer_id"`
	DepositoType   string    `json:"deposito_type"`
	DepositoTypeID int64     `json:"deposito_type_id"`
	CreatedAt      string    `json:"created_at"`
	UpdatedAt      string    `json:"updated_at"`
	Balance        string    `json:"balance"`
	BalanceRaw     float64   `json:"balance_raw"`
}

File: frontend/src/lib/api/types/account/account.ts
export interface Account {
  id: number
  account_id: string
  customer: string
  customer_id: number
  deposito_type: string
  deposito_type_id: number
  created_at: string
  updated_at: string
  balance: string
  balance_raw: number
}

export interface ListResponse<T> {
  data: T[]
  total: number
}
